pithos/plugins/mediakeys.py

```diff
@@ -68,7 +68,7 @@ def mediakey_signal(self, proxy, sender, signal, param, userdata=None):
                 self.window.playpause_notify()
             elif action == 'Next':
                 self.window.next_song()
-            elif action == 'Stop':
+            elif action == 'Stop' or 'Pause':
```

Comment:
This isn't valid syntax?

Reply:
It should be:
```elif action == 'Stop' or action == 'Pause':```

Reply:
Yes it is.

Reply:
It works as expected.

Reply:
if the action equals/is 'Stop' or 'Pause' user_pause()

Reply:
Well my bad it's not technically invalid syntax but it does not work as I expected. I'll fix it.